### DISCUSSION TITLE
Show 'new update available' notification

### DISCUSSION THREAD

**Discussion by RubenVanEldik** [nuxt/nuxt#9755] [Questions (Nuxt 2)] [answered]
Hi Nuxt.js community!

I am building a PWA with Nuxt.js and I'm planning to start rolling it out in the next few days. 😄 Since it's a new app I'll update it often and I would like to show users a notification whenever a new update is available (so they can reload the app). I've read some blog posts about how to do this for PWA's ([like this one](https://medium.com/progressive-web-apps/pwa-create-a-new-update-available-notification-using-service-workers-18be9168d717)) But I don't know what the best way to do this is in Nuxt.js. Does anybody have any experience with building such a feature?

Any help is greatly appreciated!

**Comment by TheAlexLichter** [comment, marked as answer]
Hey 👋 

Please check out the PWA module and the docs. There is a [section that showcases a similar scenario](https://pwa.nuxtjs.org/workbox/#refresh-to-update-notification).

You can combine this basic example with a banner or whatever you like 😋

**Reply by RubenVanEldik** [reply]
Oh my, I don't know how I could've missed this. This perfect, thank you Alexander! 😊
